packages/cli/src/dbt/targets/snowflake.ts

```diff
@@ -111,7 +111,6 @@ export const convertSnowflakeSchema = (
             warehouse: target.warehouse,
             database: target.database,
             schema: target.schema,
-            threads: target.threads,
```

Comment:
don't you still need to do 
`threads: target.threads`? If I'm not mistaken this will never process the input and always be default (1)

Reply:
Good point, we don't actually use `threads` in any of the adapters. So I was thinking:

1. Get rid of `threads` entirely from the interface
2. Keep `threads` in case we want it later but never use it. 

In the second case do you think it's still worth assigning the value?